code/style.css

```diff
@@ -1,5 +1,644 @@
+  transition: all 0.5s;
+  font-size: 16px;
+  margin: 5% 40%;
+}
+
+.btn:hover {
+  background-color: rgb(244, 136, 97);
+  font-weight: bold;
+}
+
+.btn:active {
+  border: rgb(244, 136, 975);
+  color: rgb(244, 136, 97);
+  background: rgb(244, 136, 97);
+}
+
+img {
+  border: 1px solid pink;
+  border-radius: 50%;
+  height: auto;
+  max-width: 300px;
+  padding: 20px;
+}
+
+video { 
+  border: 1px solid pink;
+  border-radius: 5%;
+  padding: 10px;
+  align-content: center;
+  margin: 50px 0;
+  height: auto;
+  max-width: 300px;
+}
+
+.about {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 100px 0;
+}
+
+.image_about {
+  background-image: url("/Users/rebeccabrogli/Desktop/Desktop - Rebecca’s MacBook Air/Project1/project-p1-first-webpage-master/images/rebecca.jpg");
+  background-size: contain;
+  background-color: peachpuff;
+  background-repeat: no-repeat;
+  border-radius: 5%;
+  padding: 15px;
+  border: 2px solid pink;
+  width: 480px;
+  height: 650px;
+  margin: auto;
+}
+
+.text_about {
+  display: flex;
+  flex-direction: column; 
+  flex-wrap: wrap;
+  justify-content:center;
+  margin: 100px 0;
+  background-color: peachpuff;
+  background-repeat: no-repeat;
+  border-radius: 5%;
+  padding: 15px;
+  border: 2px solid pink;
+  width: 480px;
+  height: 650px;
+  margin: auto;  
+}
+
+.image_bottom_index, .image_bottom_toolbox, .image_bottom_about {
+  background-size: cover;
+  block-size: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  max-height: 100%;
+  margin: auto;
+  height: 100px;
+  padding: 50px;
+  font-weight: lighter;
+  font-style: italic;
+  line-height: 150%;
+}
+
+.image_bottom_index {
+  background-image: url("https://images.unsplash.com/photo-1617791160505-6f00504e3519?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1828&q=80");
+}
+
+.image_bottom_toolbox {
+  background-image: url("https://images.unsplash.com/photo-1500462918059-b1a0cb512f1d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=774&q=80");
+}
+
+.image_bottom_about {
+  background-image: url("https://images.unsplash.com/photo-1444703686981-a3abbc4d4fe3?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1740&q=80");
+}
+
+form {
+  max-width: 500px;
+  text-align:center;
+  border-radius: 5%;
+  margin: 0 auto;
+}
+
+fieldset {
+  border: none; 
+  padding: 10px;
+  background: peachpuff;
+  margin: 20px;
+  border-radius: 5%;
+}
+
+label {
+  padding: 10px;
+  display:flex;
+  margin: 10px;
+  color:rgb(149, 94, 75);
+  }
+
+span {
+  display: flex;
+  align-items: center;
+  width: 20%; 
+  text-align: right;
+  margin-right: 15px;
+  flex-shrink: 0;
+}
+
+input { 
+  font-weight: 16px;
+  background-color: rgb(255, 233, 214);
+  padding: 10px;
+  margin: 10px;  
+  border-radius: 5%;
+  width: 40ch;
+  max-width: 100%;
+}
+
+textarea {
```

Comment:
I usually prevent resize of the `textarea` to not give the user the ability to break the design:

`resize: none;`